### DISCUSSION TITLE
Signing message treats as sign transaction

### DISCUSSION THREAD

**Discussion by valentyn-solana** [phantom/docs#174] [Help & Support] [answered]
### Summary

Hi I am trying to sign a message using solana/wallet-adapter and gets error: "Cannot sign transaction using sign message"

code example below, Solflare and Wallet Connect treats Uint8Array encoded message fine, so trying to figure out

### Steps to Reproduce

import { serialize } from 'borsh';
const message = new Mail({
from: new Person({
name: 'Cow',
wallet: '0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826'
}),
to: new Person({
name: 'Bob',
wallet: '0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB'
}),
contents: 'Hello, Bob!'
});
// Serialize the data
const serializedData = serialize(schema, message);
// Sign the serialized data
const signature = await walletProvider.signMessage(serializedData);

### Phantom Version

24.1.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hey @valentyn-solana,

Thank you for writing in! Do you have this app hosted somewhere? 

We haven't received any other reports of an issue like this. And message signing seems to be functioning as expected with other applications. So I would suspect it's something with your client implementation. You can see our sandbox here: https://phantom-sandboxes.vercel.app/sol-sandbox

and the code is here: https://github.com/0xproflupin/phantom-sandboxes
